.travis.yml

```diff
@@ -16,4 +16,6 @@ install:
 - bundle install
 
 script:
-- bundle exec jekyll build
+- bundle exec jekyll build 2> error.log
+- cat >&2 error.log
```

Comment:
Yeah, no difference.  Let me know if `>&2` at the end is clearer.

Thank you for the careful review!